tests/integration_python/test_main_cli.py

```diff
@@ -262,6 +264,8 @@ def test_search_matchspec(pixi: Path, dummy_channel_1: str) -> None:
         stdout_contains=["dummy-a"],
     )
 
+    verify_cli_command([pixi, "search", "python 3.12.*"])
```

Comment:
Please use a package from dummy_channel_1 or another of our test channels.
If we use the internet, our tests will become slow.

The `multiple_versions_channel_1` channel would be suitable here